scripts/export.py

```diff
@@ -92,22 +90,11 @@ def nemo_export(argv):
     nemo_in = args.source
     out = args.out
 
-    # Create a PL trainer object which is required for restoring Megatron models
```

Comment:
This is the correct way to do things - explicitly force the user to provide a Trainer - raise a error if they don't. 
Hacking commons.py::load_config_dict() is not the correct way to do this. Please revert that change.

Reply:
@ramanathan831 @borisfom

Reply:
reverted these changes here: https://github.com/NVIDIA/NeMo/pull/4039